packages/sw-build/src/lib/get-file-manifest-entries.js

```diff
@@ -2,7 +2,7 @@ const errors = require('./errors');
 const filterFiles = require('./utils/filter-files');
 const getCompositeDetails = require('./utils/get-composite-details');
 const getFileDetails = require('./utils/get-file-details');
-const getStringDetails = require('./utils/get-string-hash');
+const getStringDetails = require('./utils/get-string-details');
```

Comment:
`get-string-details` and `get-string-hash` are two distinct modules, and the reference to `get-string-hash` in the previous commit was a copy/paste error 😦